components/resources/commands.grdp

```diff
@@ -430,6 +430,9 @@
   <message name="IDS_IDC_BRAVE_SEND_TAB_TO_SELF">
     Send tab to self
   </message>
+  <message name="IDS_IDC_SHOW_WAYBACK_MACHINE_BUBBLE">
+    Show wayback machine bubble
```

Comment:
This is just the name of the view, right? Not being shown to folks?

Reply:
Updated.